packages/financial-templates-lib/test/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -101,13 +101,16 @@ contract("CryptoWatchPriceFeed.js", function() {
     networker.getJsonReturns = [...validResponses];
     await invertedCryptoWatchPriceFeed.update();
 
-    // Before period 1 should return null.
-    assert.throws(() => invertedCryptoWatchPriceFeed.getHistoricalPrice(1588376339));
+    // Before period 1 should fail.
+    await invertedCryptoWatchPriceFeed.getHistoricalPrice(1588376339).then(
+      () => assert.fail(),
+      () => {}
```

Comment:
Can you explain what's happening here?

Reply:
+ all the other assert.throws changes you made

Reply:
Ok I think I see what you're doing. Instead of returning `() => {}` can you make this more explicitly return `.catch() => assert.succeeds()` or something like that?